### PR TITLE
Add document update check item to review process

### PR DESCRIPTION
**Pull request by wuchong**
This pull request add a check item for documentation updating to the review process. And also update the documentation contribute about how to translate Chinese document.

### CODE REVIEW

**Comment by wuchong** [comment]
Hi @rmetzger  @fhueske  , would be great if you can have a look.

The [multi-language support PR](https://github.com/apache/flink/pull/7754) for flink docs is ready to be merged. But I want to postpone it a bit to wait the review process changes ready. It would be nice if we can announce the new review process (with flinkbot update) in the community once the multi-language PR merged. So that we can make the gap as small as possible.

File: contribute-documentation.zh.md [hunk 1]
```diff
@@ -39,6 +39,14 @@ cd docs
 
 如果有任何疑问，请在开发者邮件列表随时提问。
 
+## 中文文档翻译
+
+Flink 社区正在同时维护英文和中文文档。所以如果你想要更新或扩展文档，英文和中文文档都需要更新。如果你不熟悉中文，你可以创建一个用来中文文档翻译的 JIRA 并附上 `chinese-translation` 的组件名，并与当前JIRA关联起来。如果你熟悉中文，我们鼓励在一个 pull request 中同时更新两边的文档。
```
Comment (XuQianJin-Stars) [inline contribute-documentation.zh.md]: 你可以创建一个用来中文文档翻译的 JIRA 并附上 -> 你可以创建一个用于中文文档翻译的 JIRA 并附上 ?

File: contribute-documentation.zh.md [hunk 2]
```diff
@@ -39,6 +39,14 @@ cd docs
...
+
+*注意：Flink 社区目前正在翻译中文文档，有部分文档可能还未翻译。如果你正在更新的文档还未翻译，可以简单地将英文改动复制到中文文档中。*
+
+所有的中文文档位于与英文文档同名并以 `.zh.md` 后缀的文件中。你可以根据英文文档的改动来更新或扩展 `.zh.md` 文件。如果对应的 `.zh.md` 文件还未创建，你可以拷贝现有的英文文件，名称以 `.zh.md` 作为后缀。它将在 `/zh` 文件夹下生成与 html 同名的文件。
```
Comment (XuQianJin-Stars) [inline contribute-documentation.zh.md]: 它将在 `/zh` 文件夹下生成与 html 同名的文件。-> 它将在 `/zh` 文件夹下生成与英文文档同名的 html 文件。

File: reviewing-prs.zh.md [hunk 3]
```diff
@@ -91,6 +91,14 @@ title:  "如何审核 Pull Request"
 
 可以在 [Flink代码样式页面]({{ site.baseurl }}/zh/contribute-code.html#code-style) 中找到一些代码样式指南。
 
+----
+
+### 6. 英文和中文文档是否都更新了？
+
+如果这个 pull request 引入了一个新功能，该功能应该被文档化。Flink 社区正在同时维护英文和中文文档。所以如果你想要更新或扩展文档，英文和中文文档都需要更新。如果你不熟悉中文，你可以创建一个用来中文文档翻译的 JIRA 并附上 `chinese-translation` 的组件名，并与当前JIRA关联起来。如果你熟悉中文，我们鼓励在一个 pull request 中同时更新两边的文档。
```
Comment (XuQianJin-Stars) [inline reviewing-prs.zh.md]: 并与当前JIRA关联起来 -> 并与当前 JIRA 关联起来

File: contribute-documentation.md [hunk 4]
```diff
@@ -39,6 +39,14 @@ The script compiles the Markdown files into static HTML pages and starts a local
 
 Please feel free to ask any questions you have on the developer mailing list.
 
+## Chinese documentation translation
+
+The Flink community is maintaining both English and Chinese documents. So if you want to update or extend the documentation, both English and Chinese documentation should be updated. If you are not familiar with Chinese language, you can open a JIRA attached a `chinese-translation` component for Chinese documentation translation and link it with current JIRA issue. If you are familiar with Chinese language, it is encouraged to update both sides in one pull request.
```
Comment (rmetzger) [inline contribute-documentation.md]:
```suggestion
The Flink community is maintaining both English and Chinese documentation. If you want to update or extend the documentation, both English and Chinese documentation should be updated. If you are not familiar with Chinese language, you can open a JIRA with the `chinese-translation` component for Chinese documentation translation and link it with the current JIRA issue. If you are familiar with Chinese language, you are encouraged to update both sides in one pull request.
```

**Comment by rmetzger** [comment]
Regarding Flinkbot: I'm not sure if it makes sense to add a review item specific to the Chinese documentation to the bot. We do not even have any items for the documentation in general.

**Comment by wuchong** [comment]
Hi @rmetzger , thanks for the reviewing.

The review item we want to add is not specific to Chinese translation, but for documentation in general. 

In the past, we already had the problem of merging new features / API changes without updating the documentation within the same contribution. Over time, some features are missed in documentation or fixed after release. It would be much better to add a documentation check item to make features documented. The Chinese documentation check is just a supplement to the English documentation check.

File: contribute-documentation.md [hunk 5]
```diff
@@ -39,6 +39,14 @@ The script compiles the Markdown files into static HTML pages and starts a local
 
 Please feel free to ask any questions you have on the developer mailing list.
 
+## Chinese documentation translation
+
+The Flink community is maintaining both English and Chinese documentation. If you want to update or extend the documentation, both English and Chinese documentation should be updated. If you are not familiar with Chinese language, you can open a JIRA with the `chinese-translation` component for Chinese documentation translation and link it with the current JIRA issue. If you are familiar with Chinese language, you are encouraged to update both sides in one pull request.
```
Comment (fhueske) [inline contribute-documentation.md]: "... you can open a JIRA with the `chinese-translation` component ..." 
-> 
"... please open a JIRA tagged with the `chinese-translation` component ..."

File: contribute-documentation.md [hunk 6]
```diff
@@ -39,6 +39,14 @@ The script compiles the Markdown files into static HTML pages and starts a local
...
+
+*NOTE: The Flink community is translating Chinese documentations, some documents may not have been translated yet. If the document you are updating is not translated yet, you can just copy the English changes to the Chinese document.*
```
Comment (fhueske) [inline contribute-documentation.md]: "The Flink community is translating Chinese documentations" 
-> 
"The Flink community is still in the process of translating Chinese documentations"

File: contribute-documentation.md [hunk 7]
```diff
@@ -39,6 +39,14 @@ The script compiles the Markdown files into static HTML pages and starts a local
...
+
+All the Chinese documents are located in the file with the same name with English documents but in the `.zh.md` suffix. You can update or extend the `.zh.md` file according to the English documents changes. If the corresponding `.zh.md` file doesn't exited, just copy the existed English file and rename to `.zh.md` suffix. It will generate under `/zh` folder with same name as html file.
```
Comment (fhueske) [inline contribute-documentation.md]: "All the Chinese documents are located in the file with the same name with English documents but in the `.zh.md` suffix." 
-> 
"The Chinese documents are located in the same folders as the corresponding English documents. The files for both languages have the same names except that the Chinese version have the `.zh.md` suffix."

File: reviewing-prs.md [hunk 8]
```diff
@@ -91,6 +91,14 @@ This is the detailed code review of the actual changes, covering:
 
 Some code style guidelines can be found in the [Flink Code Style Page]({{ site.baseurl }}/contribute-code.html#code-style)
 
+----
+
+### 6. Are English and Chinese documentation updated?
+
+If the pull request introduces a new feature, the feature should be documented. The Flink community is maintaining both English and Chinese documents. So both English and Chinese documentation should be updated. If you are not familiar with Chinese language, you can open a JIRA attached a `chinese-translation` component for Chinese documentation translation and link it with current JIRA issue. If you are familiar with Chinese language, it is encouraged to update both sides in one pull request.
```
Comment (fhueske) [inline reviewing-prs.md]: "you can open a JIRA"
->
"please open a JIRA"

**Comment by wuchong** [comment]
Hi @XuQianJin-Stars @rmetzger @fhueske ,  thanks for your reviewing. 

I have updated the pull request and also proposed this to "[DISCUSS] Improve the flinkbot" thread.

**Comment by rmetzger** [comment]
Thanks. I'll respond in the thread.

**Comment by rmetzger** [comment]
Yes, I'm +1 on merging this PR.
I'll soon extend flinkbot to support warnings, which will include translation warnings.

**Comment by wuchong** [comment]
Thanks @rmetzger , I will soon update and merge PR https://github.com/apache/flink/pull/7754 .

**Comment by rmetzger** [comment]
Great!
I would like to make some progress on the Chinese translation of Flink :)

**Comment by wuchong** [comment]
Merging...

**Comment by wuchong** [comment]
Hi @rmetzger  @fhueske , I have merged this PR and apache/flink#7754 .  Looking forward to the flinkbot update. I think we can start the documentation translation now, and I will spread this message to some Chinese social media to attract more contributors in. 

Cheers,
Jark

**Pull request closed without merging**